### PR TITLE
Insights page

### PR DESCRIPTION
**Pull request by 1egoman**

### CODE REVIEW

File: src/helpers/auto-refresh-hoc/index.js [hunk 1]
```diff
@@ -11,29 +24,46 @@ export default function autoRefresh({interval, shouldComponentUpdate}) {
...
-        window.removeEventListener('visibilitychange', this.visiblilityChangeListener);
         window.cancelAnimationFrame(this.raf);
       }
       tick() {
-        var now = Date.now();
-        if (now - this.state.lastFrame > interval) { this.setState({ lastFrame: now }); }
+        const now = window.performance.now();
+
+        // Is the raf frame too old?
+        if (this.state.lastFrame < now - AUTO_REFRESH_OLD_RAF_DELTA_IN_MILLISECONDS) {
```
Comment (guscost) [inline src/helpers/auto-refresh-hoc/index.js]: Looks like this already includes some version of #44 ?

File: src/index.js [hunk 2]
```diff
@@ -256,6 +265,18 @@ eventSource.on('space', countChangeEvent => {
   }));
 });
 
+// If the user is logged in, sync the count of all spaces with the server every 5 minutes.
+// This is to ensure that the count on the dashboard and the actual count can't drift really far
+// apart throughout the day if you don't refresh the dashboard.
+setInterval(async () => {
```
Comment (guscost) [inline src/index.js]: Looks like #41 is covered as well

File: src/components/live-space-list/test.js [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+import sinon from 'sinon';
+
+import { LiveSpaceList } from './index';
+
+function timeout(delay) {
+  return new Promise(r => setTimeout(r, delay));
+}
+
+describe('live space list', function() {
+  it('should render list space list (smoke test)', async function() {
```
[the remark this answers is not in the document]
Comment (1egoman) [reply src/components/live-space-list/test.js]: That's the page you get to if you click `live` in the navbar.

File: src/helpers/auto-refresh-hoc/index.js (hunk 1 above)
Comment (1egoman) [reply src/helpers/auto-refresh-hoc/index.js]: Hah, yea. I remember what happened here - I made the fix in this branch, and cherry picked to a different brach so we could merge in those changes sooner. So after merging this PR, #41 and #44 should be closed.

File: src/index.js (hunk 2 above)
Comment (1egoman) [reply src/index.js]: Ditto to above

**Pull request merged**